doc/source/reference/routines.polynomials.classes.rst

```diff
@@ -187,6 +187,10 @@ Which gives the polynomial `p` in Chebyshev form. This works because
 the original polynomial. However, all the multiplications and divisions
 will be done using Chebyshev series, hence the type of the result.
 
+It is intended that all polynomial instances are immutable, therefore
+augmented operations and any other fuctionality that would violate the
```

Comment:
Might be valuable to explain this as ```augmented operations like ``+=`` ```

Reply:
Typo in `functionality` too

Reply:
I'd be happy to give a brief description of an augmented operation if you feel it is instructive. Pardon the typos, I'll correct in the morning.